src/ankh/cmd/ankh/main.go

```diff
@@ -261,3 +272,43 @@ func check(err error) {
 		os.Exit(1)
 	}
 }
+
+func runScripts(ctx *ankh.ExecutionContext, scripts []struct { Path string }) {
+	for _, script := range scripts {
+		path := script.Path
+		if path == "" {
+			log.Warnf("Missing path in script %s", script)
+			break
+		}
+		log.Infof("Running script: %s", path)
+		if ctx.DryRun == true {
+			log.Infof("- OK (dry) %s", path)
+			break
+		}
+		// pass kube context and the "global" config as a yaml environment variable
+		// also send the kube context as the first argument to the script
+		cmd := exec.Command(path, ctx.AnkhConfig.CurrentContext.KubeContext)
+		if ctx.AnkhConfig.CurrentContext.Global != nil {
+			global, err := yaml.Marshal(ctx.AnkhConfig.CurrentContext.Global)
+			check(err)
+			cmd.Env = append(
+				os.Environ(),
+				"ANKH_CONFIG_CONTEXT=" + string(global),
```

Comment:
the value of this environment value should be valid yaml. i think we need to explicitly marshal it into yaml.

Reply:
Yeah, it's being explicitly marshalled on line 292 and then passed here as a string (since it's an env variable)

Renaming to ANKH_CONFIG_GLOBAL sounds good, will do